editoast/src/views/fonts.rs

```diff
@@ -0,0 +1,91 @@
+use axum::extract::Path;
+use axum::extract::Request;
+use axum::response::IntoResponse;
+use axum::Extension;
+use editoast_authz::BuiltinRole;
+use editoast_derive::EditoastError;
+use thiserror::Error;
+use tower::ServiceExt;
+use tower_http::services::ServeFile;
+
+use crate::client::get_dynamic_assets_path;
+use crate::error::Result;
+use crate::views::AuthenticationExt;
+use crate::views::AuthorizationError;
+
+crate::routes! {
+    "/fonts/{font}/{glyph}" => fonts,
+}
+
+#[derive(Debug, Error, EditoastError)]
+#[editoast_error(base_id = "fonts")]
+enum FontErrors {
+    #[error("File '{file}' not found")]
+    #[editoast_error(status = 404)]
+    FileNotFound { file: String },
+}
+
+/// This endpoint is used by map libre to retrieve the fonts. They are separated by font and unicode block
+
+#[utoipa::path(
+    get, path = "",
+    tag = "fonts",
+    params(
+        ("font" = String, Path, description = "Requested font"),
+        ("glyph" = String, Path, description = "Requested unicode block"),
+    ),
+    responses(
+        (status = 200, description = "Glyphs in PBF format of the font at the requested unicode block"),
+        (status = 404, description = "Font not found"),
+    ),
+)]
+async fn fonts(
+    Extension(auth): AuthenticationExt,
+    Path((font, file_name)): Path<(String, String)>,
+    request: Request,
+) -> Result<impl IntoResponse> {
+    let authorized = auth
+        .check_roles([BuiltinRole::MapRead].into())
+        .await
+        .map_err(AuthorizationError::AuthError)?;
+    if !authorized {
+        return Err(AuthorizationError::Forbidden.into());
+    }
```

Comment:
Assets are open, I don't think we need to expect users to have the `MapRead` role

Reply:
This was also for the sprite assets, should I simplify both?

Reply:
We should simplify this. We can do it here or in another PR.

Reply:
I removed it for fonts.

For the sprites see https://github.com/OpenRailAssociation/osrd/pull/10828